R/fake_functions.R

```diff
@@ -9,4 +9,5 @@ fake_function <- function() {
   RSQLite::SQLite
   digest::digest
   httr2::request
+  odbc::dbConnect
```

Comment:
After removing connectivity tools, I had to add this to the fake functions to make sure the SQLite backend will still have this dependency in imports.